src/CMakeLists.txt

```diff
@@ -117,6 +117,15 @@ elseif(${PLATFORM} MATCHES "Raspberry Pi")
   set(PLATFORM_CPP "PLATFORM_RPI")
   set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
 
+  add_definitions(-D_DEFAULT_SOURCE)
```

Comment:
Why? What's missing if you remove it?

Reply:
I think I explained my findings in this conversation.
https://github.com/raysan5/raylib/pull/1229#discussion_r420404363

The sleep issue should be resolved by now (see referenced issue) https://github.com/raysan5/raylib/issues/1235

However, I could not find out what's wrong with sem_timedwait. It may be accidentally overridden somewhere.
I could not find any example makefiles for the raspberry pi requiring anything but specifying those libraries and headers to work, so I cant't explain what's wrong.
The solution works, however. That's why I proposed to stick with it for now and try to resolve this later.

Reply:
It's not as clean as I'd have liked, because there should be no reason to specify it when we are using a POSIX function like `sem_timedwait`, but ye, works is better than doesn't work.